doc/api/next_api_changes/deprecations/28074-TS.rst

```diff
@@ -0,0 +1,7 @@
+``boxplot`` and ``bxp`` *vert* parameter
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The parameter *vert: bool* and the *boxplot.vertical* rcparam has
+been deprecated on `~.Axes.boxplot` and `~.Axes.bxp`.
+It is replaced by *orientation: {"vertical", "horizontal"}* for API
+consistency.
```

Comment:
Please add the deprecatio of the rcParam here as well.

Reply:
Do you mean in the heading? or under its own heading?

Reply:
You can do it under this heading.